libyul/backends/wasm/polyfill/Interface.yul

```diff
@@ -19,14 +19,15 @@
 // NOTE: This file is used to generate `ewasmPolyfills/Interface.h`.
 
 function address() -> z1, z2, z3, z4 {
-	eth.getAddress(0:i32)
-	z1, z2, z3, z4 := mload_internal(0:i32)
+	eth.getAddress(4:i32)
+	z2, z3, z4 := mload_address(4:i32)
 }
 
 function balance(x1, x2, x3, x4) -> z1, z2, z3, z4 {
 	mstore_address(0:i32, x1, x2, x3, x4)
```

Comment:
If in other places addresses are stored at `4:i32`, then this should do the same.

Reply:
thats true :)

Reply:
There are many other helpers which have the same inconsistency (`ext*`, etc.) Please check all of them.

Reply:
I changed the layout of `mstore_address` & `mload_address` and missed to update these. The given offset will be used to store/load the address directly from. However, I think I corrected now all `mstore_address` / `mload_address` offsets.

Reply:
What did you mean with `ext*`?

Reply:
`extcodesize`, `extbalance`, etc.

Reply:
I actually think we should put addresses right aligned and make sure to zero out the top bits, e.g.:
```
    memset(0:i32, 0:i32, 12:i32) or clear_address(0:i32)
	eth.getAddress(12:i32)
	z1, z2, z3, z4 := mload_internal(0:i32)
    // or
    z2, z3, z4 := mload_address(0:i32)
```

Reason is: we want to ensure that the memory layout matches that of EVM. At least for the beginning, once we are more confident what is going on, we could revert to this option.

Reply:
Yeah that make sense. After changing this I was able to activate even more tests.